light/client.go

```diff
@@ -899,14 +899,19 @@ func (c *Client) compareFirstHeaderWithWitnesses(ctx context.Context, h *types.S
 				c.witnesses[e.WitnessIndex])
 			return err
 		case errBadWitness:
-			// If witness sent us an invalid header, then remove it. If it didn't
-			// respond or couldn't find the block, then we ignore it and move on to
-			// the next witness.
-			if _, ok := e.Reason.(provider.ErrBadLightBlock); ok {
-				c.logger.Info("Witness sent us invalid header / vals -> removing it",
-					"witness", c.witnesses[e.WitnessIndex], "err", err)
-				witnessesToRemove = append(witnessesToRemove, e.WitnessIndex)
+			// If witness sent us an invalid header, then remove it
+			c.logger.Info("witness sent an invalid light block, removing...",
+				"witness", c.witnesses[e.WitnessIndex],
+				"err", err)
+			witnessesToRemove = append(witnessesToRemove, e.WitnessIndex)
+		default: // benign errors can be ignored with the exception of context errors
```

Comment:
what do you mean as `benign` ?

Reply:
it was taken from this line https://github.com/tendermint/tendermint/commit/b69ac23fd20bdc00dea00c7c8a69fa66f2e675a9#diff-7b60ff45324e1ea4b57af7dad6b5c5387d932923a4bbd46f5323d4e4ae7257a1R1160